lib/node_modules/@stdlib/math/base/special/acoth/package.json

```
@@ -70,5 +70,90 @@
     "trigonometry",
     "radians",
     "angle"
-  ]
+  ],
+  "__stdlib__": {
+    "scaffold": {
+      "$schema": "math/base@v1.0",
+      "base_alias": "acoth",
+      "alias": "acoth",
+      "pkg_desc": "compute the inverse hyperbolic cotangent of a number",
+      "desc": "computes the inverse hyperbolic cotangent of a number",
+      "short_desc": "inverse hyperbolic cotangent",
+      "parameters": [
+        {
+          "name": "x",
+          "desc": "input value",
+          "type": {
+            "javascript": "number",
+            "jsdoc": "number",
+            "c": "double",
+            "dtype": "float64"
+          },
+          "domain": [
+            {
+              "min": "-infinity",
+              "max": "-1"
+            },
+            {
+              "min": "1",
```

```suggestion
              "max": -1
            },
            {
              "min": 1,
```

These literal values do not need to be wrapped in strings. The reason `infinity` is wrapped in a string is that it is not valid JSON as a number.